virt_lightning/shell.py

```diff
@@ -143,21 +143,26 @@ def start(configuration, context, **kwargs):
     domain = _start_domain(hv, host, context, configuration)
     if not domain:
         return
-    import time
 
-    time.sleep(4)
-    stream = conn.newStream(libvirt.VIR_STREAM_NONBLOCK)
-    console = domain.dom.openConsole(None, stream, 0)
     loop = asyncio.get_event_loop()
```

Comment:
I think this line should placed below `import libvirtaio`, no reasons init loop without using.

Reply:
There's always a consumer of `loop`, it's used to async await for `domain.reachable()`.

Reply:
https://github.com/zdykstra/virt-lightning/blob/suppress/virt_lightning/shell.py#L167-L170

Reply:
Ah, sorry didn't see in diff. This patch is fine